src/components/SvgIcon/SvgIcon.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
looks different with what i expected.  I thought there was an Icon component where yo can pass name of icon and it will render it, because there are some predefined icons in our Icon component. Just like here https://github.com/EvgeniyKvasyuk/MAHE_WP/pull/31/files#diff-71255c9b0166172a6a694813b7b28d6f4ca3328aa33d3cfe626d07d15ff5757fR17

Reply:
Updated The PR